### ISSUE TITLE
Save individuals junction track files from case config file

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#2541]
Introduce the possibility to upload individual-specific splice junction tracks when the case is loaded via yaml config file.

Check here for reference on the naming of these 2 files: https://github.com/Clinical-Genomics/scout/pull/2487

**Comment by northwestwitch** [comment]
Include splice junctions tracks when demo case is loaded as well. Test tracks from a cell line kindly provided by @jemten . They're under:

/home/proj/stage/rare-disease/cases/finequagga/analysis/ACC5963A1/build_sj_tracks/
/home/proj/stage/rare-disease/cases/sharpparrot/analysis/ACC5963A2/build_sj_tracks

**Comment by northwestwitch** [comment]
Bigwig file from the demo case in comment above looks too big to be included in the demo case (495 Mb). We should probably filter out the regions that do not contain the demo variants!

**Comment by dnil** [comment]
Filtering is fine; otw how about figshare?

**Comment by northwestwitch** [comment]
Figshare is an option, but there might be that CORS issue that you know

**Comment by dnil** [comment]
Right, but we have the built in proxy now, though perhaps we only enabled it for annotation tracks?

**Comment by northwestwitch** [comment]
> Right, but we have the built in proxy now, though perhaps we only enabled it for annotation tracks?

Yes, and what about the bigwig splice junctions tracks that are on our prod server: in that case we don't need them to be returned by the proxy endpoint

**Comment by northwestwitch** [comment]
And the proxy endpoint we have doesn't returned ranges of data, which means we need to create another specific proxy endpoint or fix the one we have. That is fine, but perhaps it's easier to filter the bigwig at this point..

**Comment by dnil** [comment]
Agreed, good points. Both solvable, but perhaps unnecessary effort right now.
